validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    TEST_SET_FILENAME = 'canonical/uc_davis_no_partition_test_random_grouped.csv' #@param
</pre><p>I saw this in Greta's notebook too. What does "no_partition" mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
It means that this is the entire dataset being used for evaluation.

Reply:
But then it also says "test". So is this the entire unpartitioned dataset or the test set?

Reply:
Entire unpartitioned dataset. If we really need to, I can change the name. But it may be a small detail if we're not planning to continue evaluation on this set altogether.

Reply:
Discussed online: This came from the request of evaluating on the training set. Now, the data ingestion script does preprocessing and grouping but no partitioning. This is a placeholder now and will be changed to the holdout set moving forward, which we have not defined yet.

Mitigation strategy: Add a TODO before submitting.